### PR TITLE
fix crash when deleting avatar-entities

### PR DESCRIPTION
**Pull request by sethalves**
- fixed a crash that happened when deleting avatar-entities

### CODE REVIEW

**Comment by sethalves** [comment]
https://highfidelity.fogbugz.com/f/cases/1971/crash-in-PhysicalEntitySimulation-handleOutgoingChanges

**Comment by sethalves** [comment]
to test:
- run this in the console:

    Entities.addEntity({
       type: "Sphere",
       color: {red: 0, green: 0, blue: 255},
       dimensions: {x: 2, y: 2, z: 2},
       position: {x: 3, y: -33000, z: 0},
       name: "off-edge",
       lifetime: 6000,
       dynamic: true,
       velocity: {x: 0, y: -1, z: 0},
       gravity: {x: 0, y: -20, z: 0 }
    }, true)

- make sure avatar-entities generally work (open market with HMD on).

**Comment by hifi-gustavo** [comment]
Build console output for commit: 9b7086b1fb3f47a7bbcc547d1cd6e4fe64e1b642 available at: https://gist.github.com/d3110dd404111a7cf8cf1eda020f2adb<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8837-9b7086b.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8837-9b7086b.dmg

File: libraries/entities/src/EntityEditPacketSender.cpp [hunk 1]
```diff
@@ -63,8 +63,8 @@ void EntityEditPacketSender::queueEditAvatarEntityMessage(PacketType type,
 
     // the properties that get serialized into the avatar identity packet should be the entire set
     // rather than just the ones being edited.
-    entity->setProperties(properties);
     EntityItemProperties entityProperties = entity->getProperties();
+    entityProperties.merge(properties);
```
Comment (sethalves) [inline libraries/entities/src/EntityEditPacketSender.cpp]: This is the actual fix -- I was using setProperties on the entity as a way to merge the entities current properties with the changed ones.  This was causing some side-effects which could cause a crash.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 5d085798cbddc54b8b922ff23707a7b06a24f55d available at: https://gist.github.com/d3110dd404111a7cf8cf1eda020f2adb<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8837-5d08579.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8837-5d08579.dmg

**Comment by thoys** [comment]
## QA Testing results

Could not reproduce crash in this build using your script. The marketplace and other clientOnly entities function fine.

QA Pass

**Pull request merged**